charts/alluxio/values.yaml

```diff
@@ -410,6 +410,8 @@ fuse:
   # - name: myConfigMapVolume
   #   mountPath: /data/configmap/myConfigMapVolume/
   volumes: []
+  updateStrategy:
```

Comment:
Please also update changelog and alluxio chart version. Thanks.

Reply:
done